### PR TITLE
Add doc for Pulsar shell clone

### PR DESCRIPTION
**Pull request by nicoloboschi**
Doc related to: https://github.com/apache/pulsar/pull/18608

### CODE REVIEW

**Comment by tisonkun** [comment]
@nicoloboschi currently, we sync doc from the main repo. Thus updates in this repo won't take effect.

Closing...

You can read the document contribution guide https://pulsar.apache.org/contribute/document-contribution for more information.

I'm trying to resolve this situation by finally host the source of doc in this site repo. But it takes time to make some prerequisite works and make a proposal :)

**Comment by tisonkun** [comment]
In short, directly update the corresponding files in the main repo (`site2/docs` folder).

**Comment by tisonkun** [comment]
Sorry..It's mid-night here and I don't find that it's about the references...

**Comment by nicoloboschi** [comment]
@tisonkun PTAL, related code change is merged now

I believe this would end up in the 2.11 doc as well, right? 🤔

**Comment by tisonkun** [comment]
> end up in the 2.11 doc

@nicoloboschi Yes. See also this comment: https://github.com/apache/pulsar/pull/16822#issuecomment-1325392898.

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 1]
```diff
@@ -68,14 +66,28 @@ Create a new configuration.
...
+
+#### `clone`
+
+Create a new configuration cloning an existing one.
+
+```bash
+default(localhost)> config clone mycluster --name mycluster2
+```
+
+
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:85]:
```suggestion

```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 2]
```diff
@@ -68,14 +66,28 @@ Create a new configuration.
...
+| Flag     | Description              | Default         |
+|----------|--------------------------|-----------------|
+| `--name` | Name of the new config.  |                 | 
+
+
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:90]:
```suggestion

```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 3]
```diff
@@ -84,7 +96,6 @@ Update an existing configuration.
 default(localhost)> config update --file ./conf/client.conf mycluster
 ```
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:98]:
```suggestion
```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 4]
```diff
@@ -100,7 +111,6 @@ Set a value for a specified configuration property.
 default(localhost)> config set-property -p webServiceUrl -v http://<cluster-hostname> mycluster
 ```
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:113]:
```suggestion
```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 5]
```diff
@@ -116,7 +126,6 @@ Get the value for a specified configuration property.
 default(localhost)> config get-property -p webServiceUrl mycluster
 ```
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:128]:
```suggestion
```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 6]
```diff
@@ -68,14 +66,28 @@ Create a new configuration.
 default(localhost)> config create --file ./conf/client.conf mycluster
 ```
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:68]:
```suggestion
```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 7]
```diff
@@ -29,7 +28,6 @@ pulsar-shell -e [COMMAND]
 echo "[COMMAND]" | pulsar-shell -
 ```
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:30]:
```suggestion
```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 8]
```diff
@@ -10,7 +10,6 @@ Usage
 pulsar-shell
 ```
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:12]:
```suggestion
```

**Review by tisonkun — APPROVED** [review]
LGTM. Several whitespace issues. Applying and merging...

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 9]
```diff
@@ -68,14 +64,23 @@ Create a new configuration.
...
+
+Create a new configuration cloning an existing one.
+
+```bash
+default(localhost)> config clone mycluster --name mycluster2
+```
+| Flag     | Description              | Default         |
+|----------|--------------------------|-----------------|
+| `--name` | Name of the new config.  |                 | 
 #### `update`
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:84]:
```suggestion
| `--name` | Name of the new config.  |                 | 

#### `update`
```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 10]
```diff
@@ -68,14 +64,24 @@ Create a new configuration.
...
 
-Options
-
 | Flag     | Description              | Default         |
 |----------|--------------------------|-----------------|
 | `--file` | File path of the config. |  | 
 | `--url`  | URL of the config.       |  |
 | `--value`  | Inline value of the config. Base64-encoded value is supported with the prefix `base64:`. |  |
 
+
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:73]:
```suggestion

```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 11]
```diff
@@ -68,13 +64,21 @@ Create a new configuration.
...
 ```
 
-Options
-
 | Flag     | Description              | Default         |
 |----------|--------------------------|-----------------|
 | `--file` | File path of the config. |  | 
 | `--url`  | URL of the config.       |  |
 | `--value`  | Inline value of the config. Base64-encoded value is supported with the prefix `base64:`. |  |
+#### `clone`
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:72]:
```suggestion

#### `clone`
```

File: site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md [hunk 12]
```diff
@@ -68,14 +64,23 @@ Create a new configuration.
...
 | `--value`  | Inline value of the config. Base64-encoded value is supported with the prefix `base64:`. |  |
 
+#### `clone`
+
+Create a new configuration cloning an existing one.
+
+```bash
+default(localhost)> config clone mycluster --name mycluster2
+```
+| Flag     | Description              | Default         |
```
Comment (tisonkun) [inline site2/website-next/static/reference/next/pulsar-shell/pulsar-shell.md:80]:
````suggestion
```

| Flag     | Description              | Default         |
````

**Pull request merged**